pybeerxml/fermentable.py

```diff
@@ -25,6 +27,18 @@ def __init__(self):
         self._yield: Optional[float] = None
         self.color: Optional[float] = None
         self._add_after_boil: Optional[bool] = None  # Should be Bool
+        self.version: Optional[int] = None
```

Comment:
Any particular reason, why this is an `int`? What about versions like `1.2` or something crazy. (I haven't really seen anyone using the `version` property anyway)

Reply:
The spec says it's an int: http://www.beerxml.com/beerxml.htm

But I wouldn't mind to make it a Text for better compatibility. What do you think?